From cf11c3d888ae49374ed2d4871560c82e4639b85b Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Sat, 29 Jan 2022 11:57:38 +0100
Subject: [PATCH] Do not use `__THROW` in prototypes

`__THROW` is a glibc-internal macro, as indicated by the
preceding double underscore. The absence of this macro breaks
builds on musl.

Bug: https://bugs.gentoo.org/828894
---
 libibverbs/neigh.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libibverbs/neigh.c b/libibverbs/neigh.c
index fa3cbf57..7eed95f1 100644
--- a/libibverbs/neigh.c
+++ b/libibverbs/neigh.c
@@ -30,7 +30,7 @@
 #if !HAVE_WORKING_IF_H
 /* We need this decl from net/if.h but old systems do not let use co-include
    net/if.h and netlink/route/link.h */
-extern unsigned int if_nametoindex(__const char *__ifname) __THROW;
+extern unsigned int if_nametoindex(__const char *__ifname);
 #endif
 
 /* for PFX */
-- 
2.35.0

